configs/pie-components-config/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@justeattakeaway/pie-components-config",
+  "version": "0.1.0",
```

Comment:
I think this will need to be `v0.0.0`, otherwise changesets will try and bump the package to `v0.2.0` when you merge the PR

Reply:
Indeed!